### PR TITLE
Bug 1861605: Update manifests to include the bug fix

### PR DESCRIPTION
**Pull request by simrandhaliw**
This PR updates manifests to include a [bug fix](https://github.com/operator-framework/api/pull/59)

### CODE REVIEW

File: deploy/ocp/manifests/0.16.0/0000_50_olm_00-catalogsources.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,201 @@
+---
```
Comment (njhale) [inline deploy/ocp/manifests/0.16.0/0000_50_olm_00-catalogsources.crd.yaml:1]: Please drop the changes to anything other than the `/manifests` directory for this patch.

Comment (simrandhaliw) [reply deploy/ocp/manifests/0.16.0/0000_50_olm_00-catalogsources.crd.yaml:1]: ok, will update the PR. Thanks!

**Pull request merged**